frontend-ssr/pages/feedback-review/[exerciseId].tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Head from 'next/head';
+
+import Header from '~/components/header/Header.component';
+import styles from '~/styles/pages/feedback-review.module.scss';
+import { Exercise } from '../api/exercises/exercise.model';
+import NotFoundPage from '../404';
+import Editor from '~/components/feedback-review/Editor';
+import Comments from '~/components/feedback-review/Comments';
+import { Comment } from '../../components/feedback-review/comment.model';
+
+interface FeedbackReviewProps {
+  exercise: Exercise;
+  comments: Comment[];
+}
+
+export default function FeedbackReview(props: FeedbackReviewProps) {
+  if (!props.exercise) return <NotFoundPage />;
+
+  return (
+    <>
+      <Head>
+        <title>Review Feedback | FrontEnd.ro</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <Header />
+      <main className={styles.page}>
+        <h1>
+          Exercitiu: <strong>{props.exercise.title}</strong>
+        </h1>
+        <p>
+          {props.exercise.description}
+        </p>
+        <Editor solution={props.exercise.solution} />
+        {/** This will need to be displayed only for admin users */}
```

Comment:
Not only admin, contributor too, right? I mean, let's say the admin gives some feedback, the contributor has some objections and comments on stuff. Right?

Reply:
Yes, I wanted to say that only the admin should see the actions of approving / rejecting. I removed the comment for now